### PR TITLE
Drop stubs compilation & Automatize protos publishing to BSR

### PR DESCRIPTION
**Pull request by altafan**
This updates the scaffolding of the project to the "standard" one we use for other projects.

This also automatize the compilation of proto stubs (golang, js, grpc-web for ts) and the publication of the proto to buf.build registry.

Please @tiero review this.

### CODE REVIEW

File: buf.gen.yaml [hunk 1]
```diff
@@ -1,27 +1,28 @@
 version: v1
+managed:
+  enabled: true
+  go_package_prefix:
+    default: github.com/vulpemventures/taxi-protobuf/api-spec/protobuf/gen
```
Comment (tiero) [inline buf.gen.yaml]: dow we need this? we will specifiy in our consumer

Comment (altafan) [reply buf.gen.yaml]: This is actually needed to compile go stubs

Comment (altafan) [reply buf.gen.yaml]: when this is merged, stubs will be automatically compiled by the gh action, and the taxi.proto published to the BSR

**Comment by altafan** [comment]
UPDATE: The original `fee_amount` field of `TopupWithAssetRequest` message have been renamed `estimated_tx_size` in order for taxi service to calculate the fee amount by multiplying it by `millisat_per_byte`.

**Review by tiero — APPROVED** [review]

**Pull request merged**